css/main.css

```diff
@@ -222,5 +224,121 @@ body {
   border-radius: 10px;
   box-shadow: 0 54px 32px #2d2e30;
 }
-.circle {
+.box-flex {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding-bottom: 108px;
+}
+.box-shadow {
+  height: 190px;
+  width: 96%;
+  background: #0d1025;
+  opacity: 0.06;
+  filter: blur(60px);
+  border-radius: 10px;
+  position: relative;
+  margin-top: 90px;
+}
```

Comment:
I think no, because our first box is shadow box and second box is white. If we want to make shadow effect, we have to positioned shadow box behind of white. It's like white box position on photo in cloud-budget project.

Reply:
Besides of it when we set absolute position for shadow box, this box shut first part of hero section.

Reply:
Try to set position absolute for box-shadow element. You need no div element. You can create this shadow effect using pseudo element for white-box.